arches/app/src/components/ControlledListManager/ItemHeader.vue

```diff
@@ -0,0 +1,62 @@
+<script setup lang="ts">
+import { inject } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import LetterCircle from "@/components/ControlledListManager/LetterCircle.vue";
+
+import { itemKey, selectedLanguageKey } from "@/components/ControlledListManager/constants.ts";
+
+import { bestLabel } from "@/components/ControlledListManager/utils.ts";
+
+import type { Ref } from "vue";
+import type { Language } from "@/types/arches";
+import type { ControlledListItem } from "@/types/ControlledListManager";
```

Comment:
Roughly, I've been doing four blocks:
```js
// import third-party, plus arches for some reason,
// maybe because the arches object is "3rd party" to the vue frontend 🤔 
import arches
import vue
import vue-gettext
// whitespace
import primevue components
// whitespace
import 1st party stuff e.g. constants, utils, other components
// but i've ended up with distracting newlines here over time, so i'll consolidate and push the components last
//whitespace
import types, in same order but all as one block
```